Fix contribution thresholds for OpenTelemetry voter eligibility

Arunodoy18 opened the pull request:
- Reduce All_All contribution threshold from 10x to 5x multiplier
- Reduce RepoGroup_All contribution threshold from 5x to 2x multiplier
- Reduce All_Country contribution threshold from 5x to 2x multiplier
- This addresses the 25-contribution cutoff issue for OpenTelemetry elections
- Now includes developers with lower activity levels for voter roll generation

Fixes: OpenTelemetry election voter roll missing contributors with <25 contributions

Please make sure that you follow instructions from [CONTRIBUTING](https://github.com/cncf/devstats/blob/master/CONTRIBUTING.md)

Specially:
- Check if all tests pass, see [TESTING](https://github.com/cncf/devstats/blob/master/TESTING.md) for deatils.
- Make sure you've added test coverage for new features/metrics.
- Make sure you have updated documentation.
- If you added a new metric, please make sure you have been following instructions about [adding new metric](https://github.com/cncf/devstats/blob/master/METRICS.md).

lukaszgryglicki commented:
LGTM but please add a documentation.
In the main README.md create a section about running in Docker.
Just put some header and a link to a new DOCKER.md file.
Put all your documentation in a new DOCER.md file.

I'll review again and merge then.

Thanks for ytour work!

lukaszgryglicki commented:
Closing this, 16.04 is no longer most up to date LTS release, let me know if you want to reopen.

Arunodoy18 committed:
Fix contribution thresholds for OpenTelemetry voter eligibility

- Reduce All_All contribution threshold from 10x to 5x multiplier
- Reduce RepoGroup_All contribution threshold from 5x to 2x multiplier
- Reduce All_Country contribution threshold from 5x to 2x multiplier
- This addresses the 25-contribution cutoff issue for OpenTelemetry elections
- Now includes developers with lower activity levels for voter roll generation

Fixes: OpenTelemetry election voter roll missing contributors with <25 contributions
Signed-off-by: Arunodoy18 <arunodoy630@gmail.com>

Arunodoy18 commented:
I have done the fixes. If any changes is needed please inform.

lukaszgryglicki commented:
Hi, thanks for this, I will get to this on my CNCF working day.
Please note that changing SQL means that given dashboard must be manually regenerated for given project(s), that means:
- I need to test if fix work first.
- - Then I need to regenerate pre-computed data for all projects that were modified.
- We're now doing a migration to new servers which may be higher priority.
- I have two weeks of vacation between Dec 4th - Dec 17th.

Just note that I will eventually work on this one, but it may take some time.
